code/game/machinery/fat_sucker.dm

```diff
@@ -182,6 +182,14 @@
 	if(occupant && iscarbon(occupant))
 		var/mob/living/carbon/C = occupant
 		if(C.type_of_meat)
+			// Someone changed component rating high enough so it requires negative amount of nutrients to create a meat slab
+			if(nutrient_to_meat < 0)
+				message_admins("[src] requires negative amount of nutrients to create a slab. Exploding it to avoid crash") // no need to actually explode it but i want to punish whoever tries to do it
```

Comment:
Don't message admins this, since it's not possible to be achieved without admin powers

Reply:
done